.devcontainer/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Based on the example NodeJS DevContainer
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0.43-10
```

Comment:
I tracked down https://github.com/microsoft/vscode-dev-containers/tree/master/containers/javascript-node-10, but I wasn't able to find where this `0.43` release is recorded or where its changelog can be found.

Reply:
Does this help? https://github.com/microsoft/vscode-dev-containers/tree/v0.43.0/containers/javascript-node-10

I just used https://github.com/microsoft/vscode-remote-try-node/blob/master/.devcontainer/Dockerfile and checked if there was a Node 10 label for the image and it worked.

Reply:
OK, yeah it looks like there is a more recent `v0.101.0` release (https://github.com/microsoft/vscode-dev-containers/tree/v0.101.0/containers/javascript-node-10). It looks like they are updating these versions pretty quickly. Does something like `*-10` work to get the latest NodeJS 10 container?

Reply:
I'm not sure. I'm pretty new to the concept myself and thought this project might be a good candidate to use it in, because of the NodeJS version requirement. The feature is still under development. I guess that's the reason for the frequent updates.

Too lax version ranges sometimes cause future problems, because a newer version is pulled that was not verified to work. That's why I often prefer to use a known-good version, even if it's a bit outdated.

Reply:
Regardless, it doesn't work:
> [2020-02-14T11:46:17.459Z] [PID 6416] [1146 ms] Step 1/5 : FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:*-10
invalid reference format

Reply:
It also appears to me that all devcontainer images are built through the same project/pipeline. So the 0.101.0 is actually the version for the parent project. The NodeJS container has only seen a handful of changes since 0.43.0: https://github.com/microsoft/vscode-dev-containers/commits/v0.101.0/containers/javascript-node-10/.devcontainer/Dockerfile

I will probably update this PR to 0.101.0 anyway though, as I like the changes that were made in the meanwhile.

Reply:
Looking at this [web page](https://hub.docker.com/_/microsoft-vscode-devcontainers?tab=description) I found that you should be able to use `mcr.microsoft.com/vscode/devcontainers/javascript-node:10`

I checked it and it worked for me

Reply:
@marosoft Thanks for the heads-up. Would you prefer I use that tag over a fixed version? I'm still slightly concerned with the instability a non-fixed version tag might introduce.

Reply:
In general, I prefer fixed versions, but perhaps, since this is fairly early on, we should go with the `:10` and document how to fix the version if you run into a version that doesn't work. We should also mention in the docs for the person to open an issue. Then we can evaluate over time if there is a need to fix the version due to breaking changes/issues.